service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/EhrExtractStatusService.java

```diff
@@ -115,13 +118,107 @@ public class EhrExtractStatusService {
     private static final String FILENAME_TYPE_PLACEHOLDER = "FILENAME_PLACEHOLDER_ID=";
     private static final String ACKS_SET = ACK_HISTORY + DOT + ACKS;
     public static final int EHR_EXTRACT_SENT_DAYS_LIMIT = 8;
-    private static final String REASON_ERROR_CODE = "04";
-    public static final String REASON_ERROR_MESSAGE = "The acknowledgement has been received after 8 days";
+    private static final String UNEXPECTED_CONDITION_ERROR_CODE = "99";
```

Comment:
As an aside, is there any documentation regarding what these error codes should be? I'd enjoy reading up on them as I don't think I've spotted anything about these so far! 😄

Reply:
I couldn't find anything in our codebase either. I think it might be somewhere described in NHS documentation.